assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -345,54 +355,69 @@ export default createWidget("discourse-reactions-actions", {
           .current_user_used_main_reaction;
         const template = document.createElement("template");
         template.innerHTML = iconHTML(
-          hasUsedMainReaction ? `far-${mainReactionIcon}` : mainReactionIcon
+          hasUsedMainReaction ||
```

Comment:
extract this please

Reply:
extract in a function outside of this scope, I want to reduce complexity of the whole function please